app/src/androidTest/java/com/sdp/swiftwallet/JavaTest/wallet/WalletInterfacesTest.java

```diff
@@ -81,6 +103,31 @@ public void shouldBeAbleToCreateIWeb3Requests(){
             public CompletableFuture<BigInteger> getBalanceOf(String hexAddress) {
                 return null;
             }
+
+            @Override
+            public CompletableFuture<Web3ResponseType> sendTransaction(String hexValue) {
+                return null;
+            }
+
+            @Override
+            public CompletableFuture<BigInteger> getChainGasPrice() {
+                return null;
+            }
+
+            @Override
+            public CompletableFuture<BigInteger> getChainGasLimit() {
+                return null;
+            }
+
+            @Override
+            public CompletableFuture<BigInteger> getAccountNonce(String hexAddress) {
+                return null;
+            }
         };
+        web3Requests.getBalanceOf("account");
```

Comment:
maybe create constants for different strings?

Reply:
Yeah right I'll make them constants when the mock web3 is entirely finished